test/run-drun/try-finally-trap/try-finally-trap.mo

```diff
@@ -0,0 +1,24 @@
+import { debugPrint; error; call_raw; principalOfActor; rts_callback_table_count } =  "mo:⛔";
+
+actor A {
+
+  public func go() : async () {
+    try {
+      await async ();
+      assert false
```

Comment:
this isn't needed. There is a trap coming up.

Reply:
Right you are.